src/components/application_manager/rpc_plugins/sdl_rpc_plugin/test/commands/hmi/rc_is_ready_request_test.cc

```diff
@@ -113,24 +113,42 @@ class RCIsReadyRequestTest
     event.set_smart_object(*msg);
   }
 
+  void InterfacesUpdateExpectations(
+      const std::vector<hmi_apis::FunctionID::eType>& interfaces_to_update) {
+    EXPECT_CALL(mock_hmi_capabilities_, GetInterfacesToUpdate())
```

Comment:
Can you pass arguments for `Times` for use `InterfacesUpdateExpectations`  in all tests in file. Try to use this one, `EXPECT_CALL(mock_hmi_capabilities_,GetInterfacesToUpdate()).Times(expectCount).WillRepeatedly(Return(interfaces_to_update));`

Reply:
@ydementieiev Done in baa43da677085c27f015a706ddd5256b4a31f17e